lib/ops/ops.go

```diff
@@ -905,6 +952,8 @@ func (s *SiteOperation) String() string {
 		typeS = "uninstall"
 	case OperationGarbageCollect:
 		typeS = "garbage collect"
+	case OperationUpdateEnvars:
+		typeS = "update cluster envars"
```

Comment:
Nit: Maybe "env vars" at least? I don't think "envars" is even a word (or a valid shorthand).